### ISSUE TITLE
Add icons to the navigation and view buttons

### ISSUE THREAD

**Issue by eppfel** [nextcloud/calendar#508]
As proposed in https://github.com/nextcloud/calendar/issues/312#issuecomment-278090752 I think it would aid the UI using icons+labels.

**Comment by georgehrke** [comment]
@eppfel Any idea where we could find some flat, public domain icons for the views?

**Comment by eppfel** [comment]
I could build them... Just wanted to have an issue, so this does not get lost.

**Comment by georgehrke** [comment]
@eppfel @nextcloud/designers Should we still pursue this idea?

**Comment by jancborchardt** [comment]
@georgehrke yep – which icons do you need? The calendars could all get a calendar icon for example. For the rest, feel free to see if [The Noun Project](https://thenounproject.com/) has some fitting ones?

**Comment by joshtrichards** [comment]
Fixed in #1579
